frontend/src/pages/NoMatchPage.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import {
+  Container,
+  Col,
+  Card,
+  CardBody,
+  CardText,
+  CardTitle
+} from 'reactstrap';
+import { Link } from 'react-router-dom';
+
+class RegisterPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      timeRemaining: 10
+    };
+  }
+
+  componentDidMount() {
+    const ten_seconds = 10000;
```

Comment:
should we have some const variable to record the starting timeRemaining (10) and to derive the amount of elapsed milliseconds before redirecting (10000)?